test/prompts.test.ts

```diff
@@ -17,10 +17,51 @@ describe("parsePromptMessage", () => {
       }),
     ).toBeUndefined();
     expect(
-      parsePromptMessage({ question: "hello?", showBefore: Date.now() }),
+      parsePromptMessage({
+        question: "hello?",
+        showBefore: Date.now(),
+        promptId: "123",
+        featureId: "123",
+      }),
+    ).toBeUndefined();
+    expect(
+      parsePromptMessage({
+        question: "hello?",
+        showAfter: Date.now(),
+        promptId: "123",
+        featureId: "123",
+      }),
+    ).toBeUndefined();
+    expect(
+      parsePromptMessage({
+        question: "hello?",
+        showAfter: Date.now(),
+        promptId: "123",
+        featureId: "123",
+      }),
     ).toBeUndefined();
```

Comment:
This seems to be a duplicate of the call before

Reply:
Thanks for noticing!